cardano-testnet/src/Testnet/Filepath.hs

```diff
@@ -40,10 +37,7 @@ makeSocketDir :: TmpAbsolutePath -> FilePath
 makeSocketDir fp = makeTmpRelPath fp </> "socket"
 
 makeTmpBaseAbsPath :: TmpAbsolutePath -> FilePath
-makeTmpBaseAbsPath (TmpAbsolutePath fp) = takeDirectory fp
+makeTmpBaseAbsPath (TmpAbsolutePath fp) = addTrailingPathSeparator $ takeDirectory fp
 
 makeLogDir :: TmpAbsolutePath -> FilePath
-makeLogDir (TmpAbsolutePath fp) = fp </> "logs"
-
-makeDbDir :: Int -> TmpAbsolutePath -> FilePath
```

Comment:
This was not used anywhere, and `"db/node-" </> show nodeNumber` part does not seem correct.